packages/koa-shopify-graphql-proxy/src/shopify-graphql-proxy.ts

```diff
@@ -1,24 +1,39 @@
 import proxy from 'koa-better-http-proxy';
 import {Context} from 'koa';
 
-export interface SessionContext extends Context {
-  session?: {accessToken?: string; shop?: string};
-}
-
 export const PROXY_BASE_PATH = '/graphql';
 export const GRAPHQL_PATH = '/admin/api/graphql.json';
 
-export default function shopifyGraphQLProxy() {
+export interface ProxyOptions {
```

Comment:
We need both or nothing right? I don't think these should be optional, rather the whole object should be.

Reply:
I was future proofing that if there are going to be other config in the future, they can be easily combined. But I agree for now it is both or nothing.